From f29ad6f0f50637f53efa2979587ba84f09037a2d Mon Sep 17 00:00:00 2001
From: KATOH Yasufumi <karma@jazz.email.ne.jp>
Date: Tue, 21 Jun 2016 11:06:25 +0900
Subject: [PATCH] plamo: Improve Plamo template

* configure to start only the minimum of service
* add ntp to ignore packages

Signed-off-by: KATOH Yasufumi <karma@jazz.email.ne.jp>
---
 templates/lxc-plamo.in | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/templates/lxc-plamo.in b/templates/lxc-plamo.in
index 48cde46..4b9e52d 100644
--- a/templates/lxc-plamo.in
+++ b/templates/lxc-plamo.in
@@ -49,7 +49,7 @@ MIRRORPATH=${MIRRORPATH:-"/pub/linux/Plamo"}
 CATEGORIES=${CATEGORIES-"00_base 01_minimum"}
 EXTRACTGRS=${EXTRACTGRS-""}
 IGNOREPKGS=${IGNOREPKGS-"grub kernel lilo linux_firmware microcode_ctl
-    cpufreqd cpufrequtils gpm"}
+    cpufreqd cpufrequtils gpm ntp kmod"}
 ADDONPKGS=${ADDONPKGS-"`echo contrib/Hamradio/{morse,qrq}`"}
 
 download_plamo() {
@@ -222,6 +222,14 @@ configure_plamo() {
   sh /tmp/netconfig.rconly
   rm -f /tmp/netconfig.rconly
   sed -i '/cmdline/s/if/& false \&\&/' $rootfs/etc/rc.d/rc.inet1.tradnet
+  # /etc/rc.d/rc.inet2
+  sed -i '/rpc.mountd/s/^/#/' $rootfs/etc/rc.d/rc.inet2
+  sed -i '/modprobe/s/^/#/' $rootfs/etc/rc.d/rc.inet2
+  # configure to start only the minimum of service
+  chmod 644 $rootfs/etc/rc.d/init.d/saslauthd
+  chmod 644 $rootfs/etc/rc.d/init.d/open-iscsi
+  rm -f $rootfs/etc/rc.d/init.d/postfix
+  rm -f $rootfs/var/log/initpkg/shadow
   return 0
 }
 
